src/commands/uplink/CorrectTime2000.ts

```diff
@@ -0,0 +1,75 @@
+import Command from '../../Command.js';
+
+import BinaryBuffer from '../../BinaryBuffer.js';
+
+
+const COMMAND_ID = 0x0c;
+const COMMAND_TITLE = 'CORRECT_TIME_2000';
+const COMMAND_BODY_SIZE = 1;
+
+
+enum CommandStatus {
+    FAIL,
+    SUCCESS
+}
+
+/**
+ * CorrectTime2000 command parameters
+ *
+ * @example
+ * {status: 1}
+ */
+interface IUplinkCorrectTime2000Parameters {
+    status: CommandStatus
+}
+
+
+/**
+ * CorrectTime2000 command
+ */
+class CorrectTime2000 extends Command {
+    constructor ( public parameters: IUplinkCorrectTime2000Parameters ) {
+        if ( !(parameters.status in CommandStatus) ) {
```

Comment:
unnecessary code, i will remove the same validation in 'uplink/SetTime2000'

Reply:
removed